server/services/user.service.js

```diff
@@ -202,24 +194,14 @@ export const assignPropertyToUser = async (toBeAssigned) => {
     throw new ErrorHandler(httpStatus.NOT_FOUND, 'No available units');
   }
 
-  const owner = await getUserById(toBeAssigned.userId).catch((error) => {
-    throw new ErrorHandler(httpStatus.INTERNAL_SERVER_ERROR, 'Internal Server Error', error);
-  });
-
   try {
     const updatedProperty = {
       id: property.id,
       $push: { assignedTo: toBeAssigned.userId },
     };
-    const updatePropertyUnit = await updateProperty(updatedProperty).catch((error) => {
+    return await updateProperty(updatedProperty).catch((error) => {
```

Comment:
I don't think it is a good idea to return await. Lets return the value of the await.
```suggestion
    const updatePropertyUnit = await updateProperty(updatedProperty).catch((error) => {
    ...
    
    return updatePropertyUnit;
```